copier/copier.go

```diff
@@ -1186,11 +1186,14 @@ func copierHandlerGet(bulkWriter io.Writer, req request, pm *fileutils.PatternMa
 								return filepath.SkipDir
 							}
 							return nil
+						} else if os.IsNotExist(err) {
+							logrus.Warningf("copier: file disappeared while reading: %q", path)
```

Comment:
Because the problem is caused by a race condition possibly/likely outside the control of the user.  Though it might make sense to have a ``--FailUnreadable`` style flag, where we throw an error (and stop) instead of a warning.  It's a tricky situation, so I was thinking maybe hold off on adding an option until/unless one is requested.  Less testing and docs to maintain that way :smile: